tasks/nunjucks.js

```diff
@@ -22,6 +22,9 @@ module.exports = function(grunt) {
         }
 
         var envOptions = { watch: false };
+        if (typeof options.autoescape === 'boolean') {
+            envOptions.autoescape = options.autoescape;
+        }
```

Comment:
We can also pass `undefined` there:

```
autoescape: options.autoescape || undefined
```

Nunjucks then will fallback to default true. But wouldn't be such solution too fragile? In theory Nunjucks shouldn't change it's behavior regarding `undefined`...